modules/ssm/outputs.tf

```diff
@@ -0,0 +1,9 @@
+output "secrets_variables" {
+  description = "The formatted secrets for the ECS task definition"
+  value       = local.secrets_variables
+}
+
+output "secret_arns" {
+  description = "The list of secret ARNs for ECS task execution role granting access"
+  value       = local.secret_arns
+}
```

Comment:
Just a question, these ARNs will be used in ECS Task Execution Policy right? Do you know what the action will be in the policy document in your case (as you're using SSM Parameter Store)? :)

Reply:
Not sure about the question, but I guess you are wondering about the policy we are going to set in the ECS Task Execution right? If that is so, the allowing action is probably `ssm:GetParameters` if I understand correctly ;)